### PR TITLE
Replacing v1 syntax

### PR DESCRIPTION
**Pull request by dockertopia**
### Proposed changes

Find/replace of `docker-compose` for `docker compose`.

### CODE REVIEW

**Comment by dockertopia** [comment]
@glours  some contexts are not exactly clear (to me yet). Especially the GPU support page.

File: compose/gpu-support.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ title: Enabling GPU access with Compose
 
 Compose services can define GPU device reservations if the Docker host contains such devices and the Docker Daemon is set accordingly. For this, make sure to install the [prerequisites](../config/containers/resource_constraints.md#gpu) if you have not already done so.
 
-The examples in the following sections focus specifically on providing service containers access to GPU devices with Docker Compose. You can use either `docker-compose` or `docker compose` commands.
+The examples in the following sections focus specifically on providing service containers access to GPU devices with Docker Compose. 
+You can use either `docker-compose` or `docker compose` commands.  
+See also, [Compose command compatibility with docker-compose](cli-command-compatibility.md).
```
Comment (glours) [inline compose/gpu-support.md:11]: Also missing for that file:
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/gpu-support.md?plain=1#L63
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/gpu-support.md?plain=1#L105
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/gpu-support.md?plain=1#L164

File: compose/extends.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ cache:
     - 6379:6379
 ```
 
-When you run `docker-compose up` it reads the overrides automatically.
+When you run `docker compose up` it reads the overrides automatically.
```
Comment (glours) [inline compose/extends.md:102]: Also missing for this file:
- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/extends.md?plain=1#L122
- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/extends.md?plain=1#L125
- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/extends.md?plain=1#L168

File: compose/index.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ anywhere.
 2. Define the services that make up your app in `docker-compose.yml`
 so they can be run together in an isolated environment.
 
-3. Run `docker compose up` and the [Docker compose command](#compose-v2-and-the-new-docker-compose-command) starts and runs your entire app. You can alternatively run `docker-compose up` using the docker-compose binary.
+3. Run `docker compose up` and the [Docker compose command](#compose-v2-and-the-new-docker-compose-command) starts and runs your entire app. You can alternatively run `docker-compose up` using Compose standalone(docker-compose binary).
```
Comment (glours) [inline compose/index.md:32]: Missing also in this file:
- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/index.md?plain=1#L135
- Not 100% for that one https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/index.md?plain=1#L140
- https://github.com/docker/docker.github.io/blob/9ecfa4627d7b7a498a4179db94e73d9eec250251/compose/index.md?plain=1#L193

File: compose/production.md [hunk 4]
```diff
@@ -60,9 +60,9 @@ recreating any services which `web` depends on.
 
 You can use Compose to deploy an app to a remote Docker host by setting the
 `DOCKER_HOST`, `DOCKER_TLS_VERIFY`, and `DOCKER_CERT_PATH` environment variables
-appropriately.
+appropriately. See also [Compose CLI environment variables](../compose/reference/envvars.md).
```
Comment (glours) [inline compose/production.md:63]: Also missing:
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/production.md?plain=1#L38
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/production.md?plain=1#L51
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/production.md?plain=1#L52

File: compose/profiles.md [hunk 5]
```diff
@@ -46,7 +46,7 @@ Here the services `frontend` and `phpmyadmin` are assigned to the profiles
 respective profiles are enabled.
 
 Services without a `profiles` attribute will _always_ be enabled, i.e. in this
-case running `docker-compose up` would only start `backend` and `db`.
+case running `docker compose up` would only start `backend` and `db`.
```
Comment (glours) [inline compose/profiles.md:49]: Missing:
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/profiles.md?plain=1#L64
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/profiles.md?plain=1#L65
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/profiles.md?plain=1#L76
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/profiles.md?plain=1#L77
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/profiles.md?plain=1#L107
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/profiles.md?plain=1#L111

File: compose/release-notes.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ redirect_from:
 - Fixed parsing of properties `deploy.limits.cpus` and `deploy.limits.pids` to respect floating-point values. Fixes [#9542](https://github.com/docker/compose/issues/9542){:target="_blank" rel="noopener" class="_"} and [#9501](https://github.com/docker/compose/issues/9501){:target="_blank" rel="noopener" class="_"}. 
 - Fixed `compose ps` output to list all exposed ports. Fixes [#9257](https://github.com/docker/compose/issues/9527){:target="_blank" rel="noopener" class="_"}.
 - Fixed spelling mistakes in `compose ps` code.
-- Fixed `docker-compose`to honor `--no-ansi` even when deprecated option is requested.
+- Fixed `docker compose`to honor `--no-ansi` even when deprecated option is requested.
```
Comment (glours) [inline compose/release-notes.md:29]: Missing:
* https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/release-notes.md?plain=1#L116

**Review by glours — CHANGES_REQUESTED** [review]
Wondering if we also should change the reference to `docker-compose.yml` by `compose.yml`? 🤔 
Maybe in a follow up PR

**Comment by dockertopia** [comment]
> Wondering if we also should change the reference to `docker-compose.yml` by `compose.yml`? 🤔 Maybe in a follow up PR

The name for the binary is also "docker-compose" by that logic (?). But I think the most confusing scenario it still the binary using v1 syntax even it's for v2 but just standalone. 

Regarding the other files, it isn't clear to me if or how it affects specification references.

File: compose/index.md (hunk 3 above)
Comment (dockertopia) [reply compose/index.md:32]: I'm also unclear if on line 140 we mean the command or the binary.

File: compose/production.md (hunk 4 above)
Comment (dockertopia) [reply compose/production.md:63]: Replaced.

File: compose/profiles.md (hunk 5 above)
Comment (dockertopia) [reply compose/profiles.md:49]: Replaced.

File: compose/release-notes.md (hunk 6 above)
Comment (dockertopia) [reply compose/release-notes.md:29]: Replaced.

File: compose/extends.md (hunk 2 above)
Comment (dockertopia) [reply compose/extends.md:102]: It didn't look there was a change needed on line 122. If so let me know. L125, L168 replaced.

File: compose/gpu-support.md (hunk 1 above)
Comment (dockertopia) [reply compose/gpu-support.md:11]: Replaced.

File: compose/extends.md (hunk 2 above)
Comment (glours) [reply compose/extends.md:102]: My bad for the line 122 🤦‍♂️
I think we should change this one

- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/extends.md?plain=1#L164

File: compose/index.md (hunk 3 above)
Comment (glours) [reply compose/index.md:32]: For the line 140, you're right it seems to mention the binary not the command

**Comment by glours** [comment]
@dockertopia  
To let people know that you're following the Compose specification we encourage user to adopt the `compose.yaml` name instead of the `docker-compose.yaml` one, that means your project can be run with any implementation of the specification

File: compose/extends.md (hunk 2 above)
Comment (dockertopia) [reply compose/extends.md:102]: Replaced.

**Comment by dockertopia** [comment]
> Wondering if we also should change the reference to `docker-compose.yml` by `compose.yml`? 🤔 Maybe in a follow up PR

Would that be a clean find/replace?

**Comment by dockertopia** [comment]
> @dockertopia To let people know that you're following the Compose specification we encourage user to adopt the `compose.yaml` name instead of the `docker-compose.yaml` one, that means your project can be run with any implementation of the specification

Created a docs ticket for this.

**Review by glours — APPROVED** [review]
LGTM

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**